### DISCUSSION TITLE
install pimcore via FTP

### DISCUSSION THREAD

**Discussion by GabrielePorro** [pimcore/pimcore#13047] [General]
Hello,
I'm trying to install pimcore but I have some problems to access the SSH on the server I have to use. Is there a zip with all the files I can upload via ftp or cpanel? 
Thank you!

**Comment by NiklasBr** [comment]
I suppose you can try to [install it locally](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html) on your machine, create a zip from that, and upload to the server. Make sure the database connection details match.
